app/screens/home/channel_list/categories_list/categories/categories.tsx

```diff
@@ -83,24 +96,43 @@ const Categories = ({categories, currentTeamId, onlyUnreads, unreadsOnTop}: Prop
         return orderedCategories;
     }, [categories, onlyUnreads, unreadsOnTop]);
 
+    const categoriesAfterLoading = useRef(categoriesToShow);
+
+    useDidUpdate(() => {
+        const t = setTimeout(() => {
+            categoriesAfterLoading.current = categoriesToShow;
```

Comment:
Categories may change for many reasons (i.e. websocket events). That is the reason we not set loading to true here. We are delaying this a bit on every change (be it a team switch or not). Since it is only 350ms, I don't think is a big problem.

Reply:
200ms should be enough